cmake/CLI11GeneratePkgConfig.cmake

```diff
@@ -0,0 +1,9 @@
+set(DEST_DIR "${CMAKE_INSTALL_PREFIX}")
+set(PACKAGE_VERSION "${VERSION_STRING}")
```

Comment:
Why not use `@PROJECT_VERSION@` (which is automatically set by CMake), and just use it inside the `.pc.in` file? And use `@ CMAKE_INSTALL_PREFIX@` too? If this is a convention, it could stay that way, but others, it seems to be simpler without adding new variables?

Reply:
Just ignorance on my part of CMake.  I've pushed a version with your suggestions.